gemrb/plugins/AREImporter/AREImporter.cpp

```diff
@@ -1209,6 +1211,7 @@ Map* AREImporter::GetMap(const char *ResRef, bool day_or_night)
 			str->ReadWord( &anim->sequence );
 			str->ReadWord( &anim->frame );
 			str->ReadDword( &anim->Flags );
+			anim->originalFlags = anim->Flags;
```

Comment:
This change broke the purple laser "doors" in the pocket plane. Scripts eventually disable them, but now saving stores the old value, so on load they're back again.

Changing this to only affect pst would help, but would leave the bug in pst. I guess the proper way to fix it at save time is to:
1. Get the changed bits between adjusted originalFlags and current Flags
2. Store originalFlags with those bits applied on top

Reply:
Your steps at the save time only work when not dealing with PST - since there will be bit diffs that we caused by `AdjustPSTFlags`. If the other games do not suffer from different bit settings, and presumably PST never changes them ingame, wouldn't it be easier just to switch between saving `originalFlags` (PST) and `Flags` (the rest)

Reply:
Unfortunately pst also modifies them, just not with AmbientActivate. But yeah, for non-pst we can just save the current ones.

Reply:
In PST, is it the portals in the maze? This was one example of on/off animations that I did not know how to trigger when quickly checking the maps.

Other than that, we need to know the bits that are touched in that case to properly write them back at the PSTish position when saving - for compatibility.

Reply:
The gears in ar0502 are one example. It's just A_ANI_ACTIVE, so we can do it like I proposed.